<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="groovy.infra.UsuarioRepositoryTeste" tests="11" skipped="0" failures="0" errors="0" timestamp="2025-04-14T01:53:59" hostname="pcosta-idea330" time="0.191">
  <properties/>
  <testcase name="deve verificar se email existe [count: 1, expected: true, #0]" classname="groovy.infra.UsuarioRepositoryTeste" time="0.014"/>
  <testcase name="deve verificar se email existe [count: 0, expected: false, #1]" classname="groovy.infra.UsuarioRepositoryTeste" time="0.002"/>
  <testcase name="deve salvar um usuário com sucesso" classname="groovy.infra.UsuarioRepositoryTeste" time="0.028"/>
  <testcase name="deve lançar exceção e fazer rollback ao falhar ao salvar usuário" classname="groovy.infra.UsuarioRepositoryTeste" time="0.013"/>
  <testcase name="deve listar usuário por ID com sucesso" classname="groovy.infra.UsuarioRepositoryTeste" time="0.019"/>
  <testcase name="deve retornar null ao buscar usuário por ID inexistente" classname="groovy.infra.UsuarioRepositoryTeste" time="0.009"/>
  <testcase name="deve listar todos os usuários com sucesso" classname="groovy.infra.UsuarioRepositoryTeste" time="0.021"/>
  <testcase name="deve editar usuário com sucesso" classname="groovy.infra.UsuarioRepositoryTeste" time="0.048"/>
  <testcase name="deve retornar false ao tentar editar usuário inexistente" classname="groovy.infra.UsuarioRepositoryTeste" time="0.007"/>
  <testcase name="deve excluir usuário com sucesso" classname="groovy.infra.UsuarioRepositoryTeste" time="0.008"/>
  <testcase name="deve retornar false ao tentar excluir usuário inexistente" classname="groovy.infra.UsuarioRepositoryTeste" time="0.006"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
